Dockerfile

```diff
@@ -139,7 +139,25 @@ RUN apt-get update -y && \
     vim \
     psmisc \
     qpdf \
-    tree
+    tree \
+    # pyenv dependencies
+    make \
+    build-essential \
+    libssl-dev \
+    zlib1g-dev \
+    libbz2-dev \
+    libreadline-dev \
+    libsqlite3-dev \
+    wget \
+    curl \
```

Comment:
Are some of these duplicates? I'm almost in favor of just merging the lists together. In vim you can do:
```
CMD+SHIFT+v # visual select
select whole list w/ j & k
type :sort ui
```